spec/controllers/tasks_controller_spec.rb

```diff
@@ -736,6 +736,28 @@
         expect(JSON.parse(response.body, symbolize_names: true)).to eql({status: 'failure', message: 'You need to choose a file.'})
       end
     end
+
+    context 'when service throws version not supported error' do
+      before do
+        allow(ProformaService::CacheImportFile).to receive(:call).and_raise(ProformaXML::ProformaError.new(['version not supported']))
+      end
+
+      it 'renders correct json' do
```

Comment:
```suggestion
      it 'renders correct JSON' do
```

Nitpick. JSON is an acronym. But you probably use `json` everywhere.